testing.rb

```diff
@@ -0,0 +1,81 @@
+# This is the Styleguide `HYDRATION` File
```

Comment:
we will want to move this into the kss folder maybe? Or move this code into the rake task. I think the rake task is the best spot.
— @DTrejo